### PR TITLE
use elevation instead of shadows on Android

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Comment by osdnk** [comment]
image
image
image

**Review by brunobar79 — APPROVED** [review]

File: src/components/token-family/TokenFamilyWrap.js [hunk 1]
```diff
@@ -72,9 +72,6 @@ export default function TokenFamilyWrap({
           title={title}
         />
       ) : null}
-      {/*
```
Comment (mikedemarais) [inline src/components/token-family/TokenFamilyWrap.js:75]: 💘

File: src/react-native-shadow-stack/ShadowStack.js [hunk 2]
```diff
@@ -46,10 +46,14 @@ const ShadowStack = React.forwardRef(
         width={width}
         zIndex={1}
       >
-        {shadows?.map(renderItem)}
+        {ios && shadows?.map(renderItem)}
         <View
           {...props}
           borderRadius={borderRadius}
+          elevation={shadows.reduce(
+            (acc, curr) => acc + Math.min(6, curr[2]),
```
Comment (mikedemarais) [inline src/react-native-shadow-stack/ShadowStack.js:54]: @osdnk should this be memoized?

File: src/react-native-shadow-stack/ShadowView.js [hunk 3]
```diff
@@ -1,22 +1,17 @@
-import React, { useMemo } from 'react';
+import React from 'react';
 import { StyleSheet, View } from 'react-native';
-import { Shadow } from 'react-native-neomorph-shadows';
 
 const ShadowView = props => {
   if (ios || props.elevation > 0) {
     return <View {...props} />;
+  } else {
```
Comment (mikedemarais) [inline src/react-native-shadow-stack/ShadowView.js:7]: super minor, but this `else` is unnecessary 

not reallyyyy necessary for you to change tho, its super minor

**Review by mikedemarais — APPROVED** [review]
left minor comments, but if @osdnk disagrees with them then this LGTM

**Pull request merged**